data/src/main/java/ca/bc/gov/data/datasource/local/dao/VaccineRecordDao.kt

```diff
@@ -44,6 +44,6 @@ interface VaccineRecordDao {
     @Query("SELECT * FROM vaccine_dose WHERE vaccine_record_id = :vaccineRecordId")
     suspend fun getVaccineDoses(vaccineRecordId: Long): List<VaccineDoseEntity>
 
-    @Query("DELETE FROM vaccine_record WHERE patient_id = :patientId AND data_source = 'BCSC'")
+    @Query("DELETE FROM vaccine_record WHERE patient_id = :patientId")
     suspend fun deleteAllAuthenticatedVaccineRecord(patientId: Long): Int
```

Comment:
rename method to delete()

Reply:
renamed to deletePatientVaccineRecords() as delete already exists